xbmc/interfaces/json-rpc/PlaylistOperations.cpp

```diff
@@ -212,6 +212,87 @@ JSONRPC_STATUS CPlaylistOperations::Swap(const std::string &method, ITransportLa
   return ACK;
 }
 
+JSONRPC_STATUS CPlaylistOperations::SetShuffle(const std::string &method, ITransportLayer *transport, IClient *client, const CVariant &parameterObject, CVariant &result)
+{
+  int playlist = GetPlaylist(parameterObject["playlistid"]);
+  CVariant shuffle = parameterObject["shuffle"];
+  CGUIWindowSlideShow *slideshow = NULL;
```

Comment:
I know this has been copied from another method but if you put braces around the `case PLAYLIST_PICTURE:` block you can move the definition of `CGUIWindowSlideShow *slideshow` into that block as well.

Reply:
It's done.